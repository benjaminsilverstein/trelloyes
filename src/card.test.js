import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card.js';

describe('card component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Card title="Hello" content="lorem ipsum"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });

  it('renders the UI as expected with no unreads', () => {
    const tree = renderer
      .create(<Card title="Hello" content="lorem ipsum"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
});