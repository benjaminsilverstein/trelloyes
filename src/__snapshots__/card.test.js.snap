// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card component renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button>
    delete
  </button>
  <h3>
    Hello
  </h3>
  <p>
    lorem ipsum
  </p>
</div>
`;

exports[`card component renders the UI as expected with no unreads 1`] = `
<div
  className="Card"
>
  <button>
    delete
  </button>
  <h3>
    Hello
  </h3>
  <p>
    lorem ipsum
  </p>
</div>
`;
